### ISSUE TITLE
Add collaborators to the project

### ISSUE THREAD

**Issue by fulldecent** [mleibman/SlickGrid#1163]
Please add everyone that made commits to this project as a collaborator. Then also add an official project scope to describe which contributions should and should not be accepted.

For this one-time effort you will allow others to pick up where you left off. And you allow your project to continue without any remaining effort from you.

**Comment by pmorch** [comment]
I propose modifying the title of this issue to "Transfer project to an organization".
https://help.github.com/articles/creating-a-new-organization-from-scratch/

It works really well. As the original author, you create a new organization and transfer the existing repository (including all metadata such as issues, wiki, pull requests, etc.) to the newly created organization. You can then add the people that are most active anyway as admins, committers, co-owners or whatever it is called. That way there is a single authoritative source and issue list and people can collaborate.

**Comment by 6pac** [comment]
At the moment, the project has moved to the https://github.com/6pac/SlickGrid/ repo.
This repo is unmaintained and pretty much dead.

I've been doing maintenance solo, but I did add my first collaborator recently, someone who consistently delivered high quality and well thought out PRs. I'm very reluctant to add too many collaborators. In my experience a lot of people want the glory but can't back it up with the necessary skills. That may sound harsh, but I think a lot of open source projects probably have the same experience.

**Comment by fulldecent** [comment]
Got it. Thank you I'll send new issues there.

Damn, I just found out that Google abandoned the Realtime API. SlickGrid + Realtime API was going to be a full and customizable Google Sheets alternative that also works on Google Drive. That sucks.

**Comment by 6pac** [comment]
Yeah, looks like there's no real alternative, either. Not sure why they'd deprecate such a unique and useful API.

Gotta say though, that's why I'm always slow to take up new frameworks. Seen too many of em come and go, even just over the course of a few years.

**Comment by 6pac** [comment]
@fulldecent Don't know if you've checked, but there's a bunch of stuff happening over on the 6pac repo these days, if you're interested.
